presto-main/src/test/java/com/facebook/presto/security/TestAccessControlManager.java

```diff
@@ -89,30 +92,31 @@ public void testReadOnlySystemAccessControl()
         QualifiedObjectName tableName = new QualifiedObjectName("catalog", "schema", "table");
         TransactionManager transactionManager = createTestTransactionManager();
         AccessControlManager accessControlManager = new AccessControlManager(transactionManager);
+        AccessControlContext ctx = new AccessControlContext(new QueryId(QUERY_ID), Optional.empty());
```

Comment:
nit: We do our best to not to use abbreviation int the code base. You can use `context` instead.